src/lib/types/Events.ts

```diff
@@ -57,26 +57,38 @@ export enum Events {
 	// #endregion Discord.js base events
 
 	// #region Sapphire load cycle events
-	Unload = 'unload',
-	PostLoad = 'postLoad',
+	PieceUnload = 'pieceUnload',
+	PiecePostLoad = 'piecePostLoad',
 	MentionPrefixOnly = 'mentionPrefixOnly',
+	EventError = 'eventError',
 	PrefixedMessage = 'prefixedMessage',
 	UnknownCommandName = 'unknownCommandName',
 	UnknownCommand = 'unknownCommand',
-	PreCommandRun = 'preCommandRun'
+	PreCommandRun = 'preCommandRun',
+	CommandDenied = 'commandDenied',
+	CommandAccepted = 'commandAccepted',
```

Comment:
Do we need an event between precondition success and run?

Reply:
Yes.